lib/config/typescript-prettier.js

```diff
@@ -1,19 +1,4 @@
 module.exports = {
+  // DEPRECATED: plugin:shopify/prettier is sufficient now
```

Comment:
`DEPRECATED: Please use plugin:shopify/prettier.  plugin:shopify/typescript-prettier shall be removed in a future major release` to  lay out a future plan perhaps?